Frameworks/Foundation/NSOperation.mm

```diff
@@ -277,35 +244,35 @@ - (void)main {
  @Status Interoperable
 */
 - (void)waitUntilFinished {
-    pthread_mutex_lock(&priv->finishLock);
+    pthread_mutex_lock(&_finishLock);
     while (![self isFinished]) {
-        pthread_cond_wait(&priv->finishCondition, &priv->finishLock);
+        pthread_cond_wait(&_finishCondition, &_finishLock);
     }
-    pthread_mutex_unlock(&priv->finishLock);
+
+    pthread_mutex_unlock(&_finishLock);
 }
 
 /**
  @Status Interoperable
 */
-- (id)dependencies {
-    return priv->dependencies;
+- (NSArray*)dependencies {
```

Comment:
[self willChangeValueForKey:@"dependencies"];
    [self didChangeValueForKey:@"dependencies"];
added to addDependency and removeDependency

---
In reply to: [69357321](https://github.com/Microsoft/WinObjC/pull/602#discussion_r69357321) [](ancestors = 69357321)